cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -414,6 +414,14 @@
   </property>
 
   <property>
+    <name>apps.update.schedules</name>
+    <value>true</value>
+    <description>
+      Enable if changes in schedules of an application can be updated on app redeploy.
```

Comment:
Need to discuss this, but I think this might be better:
```
If true, redeploying an application will update any schedules that currently exist for the application; if false, existing
schedules are maintained without change